doc.go

```diff
@@ -125,6 +125,8 @@ A complete connection string looks similar to the following:
 Session-level parameters can also be set by using the SQL command "ALTER SESSION"
 (https://docs.snowflake.com/en/sql-reference/sql/alter-session.html).
 
+Alternatively, use OpenWithConfig() function to create a database handle with the specified Config.
```

Comment:
We should add a test that uses OpenWithConfig() just like a user of the library would. Essentially testing the publicly exposed interface.

Reply:
added test